portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsDM.testcase

```diff
@@ -213,6 +213,47 @@ definition {
 			changeUser = "Test Test");
 	}
 
+	@description = "This is the use case for LPS-199845. Deletion Modification Conflict for DDM Structures should not be edited"
+	@priority = 3
+	test DeletionModificationConflictForDDMStructuresShouldNotBeEdited {
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(structureName = "WC Structure Title");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Text",
+			fieldName = "Text");
+
+		WebContentStructures.saveCP();
+
+		JSONPublications.addPublication(publicationName = "Publication Name");
+
+		JSONPublications.selectPublication(publicationName = "Publication Name");
+
+		WebContentNavigator.gotoEditStructure(structureName = "WC Structure Title");
+
+		WebContentStructures.addName(structureName = "WC Structure Title Edit");
+
+		WebContentStructures.saveCP();
+
+		Publications.workOnProduction();
+
+		WebContentStructures.deleteCP(structureName = "WC Structure Title");
+
+		PublicationsNavigator.openPublicationsAdmin();
+
+		PublicationsNavigator.gotoPublicationEllipsisItem(
+			menuItem = "Publish",
+			publicationName = "Publication Name");
+
+		AssertElementPresent(
+			key_changeCategory = "Dynamic Data Mapping Structure Layout",
+			key_changeTitle = "Dynamic Data Mapping Structure Layout",
+			locator1 = "PublicationsChanges#DISCARD_CHANGE_BUTTON");
+	}
```

Comment:
Can you add a step to assert there is no “Edit“ option displayed next to the DDM Structure?

Reply:
> Can you add a step to assert there is no “Edit“ option displayed next to the DDM Structure?

Add step assert `PUBLICATIONS_CHANGE_DROPDOWN` element is not present